main.go

```diff
@@ -71,11 +71,11 @@ const (
 )
 
 const (
-	defaultWebhookPort = 9443
-	defaultMetricsAddr = ":8080"
+	defaultWebhookPort    = 9443
+	defaultMetricsAddr    = ":8080"
 	defaultMetricsEnabled = true
-	defaultProbeAddr   = ":8081"
-	defaultNamespace   = "numaresources-operator"
+	defaultProbeAddr      = ":8081"
+	defaultNamespace      = "numaresources-operator"
```

Comment:
where do we use these?

Reply:
I only added `defaultMetricsEnabled` which is used [here](https://github.com/rbaturov/numaresources-operator/blob/enable-prometheus/main.go#L133)